ci/package_list.sh

```diff
@@ -12,8 +12,8 @@ RELEASE_PACKAGES=(
 )
 export RELEASE_PACKAGES
 
-# Deprecated packages are still built but not explicitly tested
-# This allows users to still access the packages if needed, however
-# it is only reserved for packages with a more public facing API
-DEPRECATED_PACKAGES=()
-export DEPRECATED_PACKAGES
+# List of binaries which should be built
+BINARIES=(
+    tedge
```

Comment:
Why is `tedge-p11-server` not listed here? Is `tedge-p11-server` supposed to be released as one package?

Reply:
no, the tedge-p11-server will be added in the next PR which adds the additional packaging files (as it is useless just to build the tedge-p11-server binary without packaging it with system service/socket etc.)